searchlib/src/vespa/searchlib/attribute/imported_search_context.cpp

```diff
@@ -160,8 +167,16 @@ void ImportedSearchContext::makeMergedPostings()
                                               committedTargetDocIdLimit));
     _merger.reserveArray(targetResult.weightedRefs.size(), targetResult.sizeSum);
     const auto &reverseMapping = _reference_attribute.getReverseMapping();
+    if (isFilter) {
```

Comment:
Bitvector is not the most efficient alternative if the synthetic posting list is small.

Reply:
No, that is correct. But I guess we might have a cheap approximation that might give a lowerbound.